cli/tests/unit/lock_test.ts

```diff
@@ -0,0 +1,161 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+import { assertEquals, unitTest } from "./test_util.ts";
+import { readAll } from "../../../test_util/std/io/util.ts";
+
+unitTest(
```

Comment:
Just a note that these tests are flaky. Perhaps we could update these tests to use the changes in:

https://github.com/denoland/deno/pull/12255/files

Then remove that file and only use this one (since this transitively tests those functions).